lib/package/config.coffee

```diff
@@ -150,12 +150,178 @@ config =
+                type: 'string'
+                enum: ['no split', 'left', 'up', 'right', 'down']
+                default: 'no split'
+                radio: true
+                order: 2
+          plotPane:
+            title: 'Plot Pane'
+            type: 'object'
+            order: 5
+            collapsed: true
+            properties:
+              defaultLocation:
+                title: 'Default location of Plot Pane'
+                type: 'string'
+                enum: ['center', 'left', 'bottom', 'right']
+                default: 'center'
+                radio: true
+                order: 1
+              split:
+                title: 'How to split Plot Pane'
+                type: 'string'
+                enum: ['no split', 'left', 'up', 'right', 'down']
+                default: 'right'
+                radio: true
+                order: 2
+          debuggerPane:
+            title: 'Debugger Pane'
+            type: 'object'
+            order: 6
+            collapsed: true
+            properties:
+              defaultLocation:
+                title: 'Default location of Debugger Pane'
+                type: 'string'
+                enum: ['center', 'left', 'bottom', 'right']
+                default: 'right'
+                radio: true
+                order: 1
+              split:
+                title: 'How to split Debugger Pane'
+                type: 'string'
+                enum: ['no split', 'left', 'up', 'right', 'down']
+                default: 'no split'
+                radio: true
+                order: 2
+          profiler:
+            title: 'Profiler'
+            type: 'object'
+            order: 7
+            collapsed: true
+            properties:
+              defaultLocation:
+                title: 'Default location of Profiler Pane'
+                type: 'string'
+                enum: ['center', 'left', 'bottom', 'right']
+                default: 'center'
+                radio: true
+                order: 1
+              split:
+                title: 'How to split Profiler Pane'
+                type: 'string'
+                enum: ['no split', 'left', 'up', 'right', 'down']
+                default: 'right'
+                radio: true
+                order: 2
+          linter:
+            title: 'Linter'
+            type: 'object'
+            order: 8
+            collapsed: true
+            properties:
+              defaultLocation:
+                title: 'Default location of Linter Pane'
+                type: 'string'
+                enum: ['center', 'left', 'bottom', 'right']
+                default: 'bottom'
+                radio: true
+                order: 1
+              split:
+                title: 'How to split Linter Pane'
+                type: 'string'
+                enum: ['no split', 'left', 'up', 'right', 'down']
+                default: 'no split'
+                radio: true
+                order: 2
       noAutoParenthesis:
```

Comment:
I think this setting is kind of `Julia Options` rather than `UI Options`.
Stays here for consistency for the past version ?
@pfitzseb

Reply:
Yeah, probably makes sense to move it. Should probably update [`updateSettings`](https://github.com/JunoLab/atom-julia-client/blob/master/lib/package/settings.js) to be recursive and clean out the old config (there's a problem where old settings are still shown if the user has changed it).

Reply:
Ah I've not noticed the file. Will take a look at and modify it.

Reply:
@pfitzseb I've implemented recursive version of `updateSettings` 😉